ibft/controller/controller_sync.go

```diff
@@ -33,9 +34,18 @@ func (i *Controller) processSyncQueueMessages() {
 
 // ProcessSyncMessage processes sync messages
 func (i *Controller) ProcessSyncMessage(msg *network.SyncChanObj) {
+	i.logger.Info("ProcessSyncMessage: processing message")
 	var lastChangeRoundMsg *proto.SignedMessage
 	currentInstaceSeqNumber := int64(-1)
 	if i.currentInstance != nil {
+		if i.currentInstance.State() == nil {
```

Comment:
Would be better to get state once (`state := i.currentInstance.State()`) and then check that variable.

Reply:
@amir-blox Thanks, good idea

Reply:
I have removed the debug logs, so the amount of usages of `i.currentInstance.State()` now is the same as it was before the PR (3). But if I change the code so that it gets state once, I'll need to add an extra nested `if`, so I think we can leave it as it was if we don't keep debug logs

Reply:
done